Makefile

```diff
@@ -22,7 +22,19 @@ AR= arm-linux-gnueabihf-ar
 CFLAGS += -march=armv7 -mthumb
 endif
 
-all:capture startrails keogram
+
+all:capture startrails keogram sunwait-remove-precompiled sunwait
+
+sunwait-remove-precompiled:
+ifneq ("arm", $(findstring $(platform), "arm"))
+	@rm -f sunwait
+endif
+
+sunwait:
+		git submodule init
+		git submodule update
+		$(MAKE) -C sunwait-src
```

Comment:
I have tried a fresh install of allsky using the `--recursive` argument but it didn't create the `sunwait-src` directory. As a temporary workaround, I have included it to the repo.

Reply:
Harrrumph.

I'll take another look at this to try figure out what went wrong.